frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -209,224 +209,201 @@ function roleSelected(evt: any) {
                                 * userId field is entered.
                                 * userId entered does not contains basic validation errors.
                             -->
-                            <div
-                                class="col-md-2"
-                                v-if="
-                                    formData.domain === domainOptions.IDIR &&
-                                    formData.userId &&
-                                    errors.userId == undefined
+                        <div
+                            class="col-md-2"
+                            v-if="
+                                formData.domain === domainOptions.IDIR &&
+                                formData.userId &&
+                                errors.userId == undefined
+                            "
+                        >
+                            <Button
+                                class="button p-button-tertiary p-button-outlined"
+                                aria-label="Verify user IDIR"
+                                :label="buttonLabel"
+                                @click="
+                                    verifyIdentity(
+                                        formData.userId,
+                                        formData.domain
+                                    )
                                 "
+                                :disabled="loading"
                             >
-                                <Button
-                                    class="button p-button-tertiary p-button-outlined"
-                                    aria-label="Verify user IDIR"
-                                    :label="buttonLabel"
-                                    @click="
-                                        verifyIdentity(
-                                            formData.userId,
-                                            formData.domain
-                                        )
-                                    "
-                                    :disabled="loading"
-                                >
-                                </Button>
-                            </div>
+                            </Button>
                         </div>
                     </div>
 
                     <div class="row" v-if="verifiedUserIdentity">
-                        <div class="form-group col-md-5 px-0">
+                        <div class="col-md-5 px-0">
                             <UserIdentityCard
                                 :userIdentity="verifiedUserIdentity"
                             ></UserIdentityCard>
                         </div>
                     </div>
 
-                    <div class="form-group col-md-3 px-0">
-                        <label for="roleSelect" class="control-label px-0"
+                    <div class="row">
+                        <label
```

Comment:
The primevue dropdown component doesn't support the id well, even I tried to give an id to the dropdown component, the label still shows the error message. Will remove the "for" attribute, just leave it as this.

Reply:
Removing it will not have an impact on accessibility?

Reply:
should be okay